lxd/device/device_runconfig.go

```diff
@@ -0,0 +1,14 @@
+package device
+
+// RunConfigItem represents a single config item.
+type RunConfigItem struct {
+	Key   string
+	Value string
+}
+
+// RunConfig represents LXD defined run-time config used for device setup.
+type RunConfig struct {
+	NetworkInterfaces [][]RunConfigItem
```

Comment:
This is effectively the same as a `[]map[string]string` as used by the others, any reason for the inconsistency?

Reply:
@stgraber I originally used a `map[string]string` here, like the others, but found there was a need to add multiple LXC items of the same key value (such as multiple `ipv4.address` lines when expanding them from the comma delimited list LXD takes). I may need to use this in the future for the other device types.

Reply:
Ah okay